src/api/spec/factories/packages.rb

```diff
@@ -3,7 +3,12 @@
     sequence(:name) { |n| "#{Faker::Internet.domain_word}#{n}" }
     factory :package_with_file do
       after(:create) do |package|
-        Suse::Backend.put("/source/#{package.project.name}/#{package.name}/somefile.txt", Faker::Lorem.paragraph)
```

Comment:
I removed the global write through enablement and added a comment that this has to be turned on when creating new factories, or updating existing